### DISCUSSION TITLE
How to get Images / Files inside collection

### DISCUSSION THREAD

**Discussion by cpt-n3mo** [directus/directus#8306] [Archive]
Good day,!

Im new to Directus and as of what i have seen now im very impressed and i like what im seeing..
im just struggling abit with the relations.. for instance i have the following setup;
1 collection, col_0 

`
col_0 contains the following:
product,
price,
images : M2M | multiple files
`

`
/items/col_0?fields=*.*
`

returns  product and price,   but not images,
how do i return these images relations..?

**Comment by rijkvanzanten** [comment]
A many-to-many field is nothing more than a one-to-many to a junction table, followed by a many-to-one to the related collection. In this case, you need to go an extra level deep, as it's `col_0` -> `col_0_directus_files` -> `directus_files`

Try `/items/col_0?fields=*.*.*`

**Reply by cpt-n3mo** [reply]
Good day..! 
Thank you for your response.
i tried that as well but this gives me the same as i would get with ` *.*`
so i figured something went wrong with the setup, so i removed all tables and started over.. apparently i removed one of the junction tables. and this caused the problem .. it works now like a charm so thank you very match !
